.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -0,0 +1,124 @@
+name: unit_test_CI
+# runs unit tests on AMD64 machine
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      mainline
```

Comment:
These test should only be run manually, rather than on every push to mainline

Reply:
revised